src/Widgets/NowPlayingWidget.m

```diff
@@ -78,23 +78,38 @@ - (void)viewDidDisappear
 
 - (void)resetNowPlaying
 {
-    NSImage *icon = [NowPlaying sharedInstance].appIcon;
+    NSImage *appIcon = [NowPlaying sharedInstance].appIcon;
+    NSString *appName = [NowPlaying sharedInstance].appName;
     NSString *title = [NowPlaying sharedInstance].title;
     NSString *subtitle = [NowPlaying sharedInstance].artist;
+    NSImage *albumArt = [NowPlaying sharedInstance].albumArt;
 
-    if (nil == icon && nil == title && nil == subtitle)
+    if (nil == appIcon && nil == title && nil == subtitle)
+    {
         title = @"♫";
-
+    }
+    else if (nil == title && nil == subtitle)
+    {
+        title = appName;
+    }
+    
     ImageTitleViewLayoutOptions layoutOptions = 0;
-    if (nil != icon)
+    if (nil != appIcon || nil != albumArt)
         layoutOptions = layoutOptions | ImageTitleViewLayoutOptionImage;
     if (nil != title)
         layoutOptions = layoutOptions | ImageTitleViewLayoutOptionTitle;
     if (nil != subtitle)
         layoutOptions = layoutOptions | ImageTitleViewLayoutOptionSubtitle;
 
     NowPlayingWidgetView *view = self.view;
-    view.image = icon;
+    if (nil != albumArt)
```

Comment:
Assuming that we can get `albumArt` to work, we should have a user setting that determines whether the `albumArt` or the `appIcon` is shown. We should not just default to the `albumArt.`